reference/requirements.rst

```diff
@@ -6,57 +6,30 @@
 Requirements for Running Symfony
 ================================
 
-To run Symfony, your system needs to adhere to a list of requirements. You
-can easily see if your system passes all requirements by running the
-``web/config.php`` in your Symfony distribution. Since the CLI often uses
-a different ``php.ini`` configuration file, it's also a good idea to check
-your requirements from the command line via:
+Symfony 2.7 requires **PHP 5.3.9** or higher to run, in addition to other minor
+requirements. To make things simple, Symfony provides a tool to quickly check if
+your system meets all those requirements.
 
-.. code-block:: terminal
-
-    $ php app/check.php
-
-Below is the list of required and optional requirements.
-
-Required
---------
+Beware that PHP can define a different configuration for the command console and
+the web server, so you need to check requirements in both environments.
 
-* PHP needs to be a minimum version of PHP 5.3.9
-* `JSON extension`_ needs to be enabled
-* `ctype extension`_ needs to be enabled
-* Your ``php.ini`` needs to have the ``date.timezone`` setting
+Checking Requirements for the Web Server
+----------------------------------------
 
-.. caution::
+Browse the ``web/config.php`` file in your browser and fix the reported issues.
+When using the PHP built-in web server, this file is available at
```

Comment:
Just spotted this one: talking about the built-in web server here IMO does not make sense. The built-in web server is started using the same configuration that is used when running the CLI check below. So that's just doing work twice. What is actually important is that you perform the check using your "real" web server setup (so most likely Nginx or Apache).